game/entities.py

```
@@ -0,0 +1,252 @@
+            width (int): The width of the terrain (screen width)
+            height (int): The height of the terrain (screen height)
+        """
+        self.width = width
+        self.height = height
+        self.heights = []
+        self.landing_zones = []
+        
+        # Generate the terrain
+        self._generate_terrain()
+    
+    def _generate_terrain(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_generate_terrain`

Add return type annotation: `None`

(ANN202)